tools/clang/test/CMakeLists.txt

```diff
@@ -111,3 +116,15 @@ if (WIN32)
           )
 endif()
 # HLSL Change End
+
+# HLSL Change Begin - Add cmds tests
+if (WIN32)
```

Comment:
We shouldn't add an explicit target for this either. #4801, will add test targets for every subdirectory without needing to manually add them.